pandas/tests/frame/methods/test_replace.py

```diff
@@ -1659,3 +1659,17 @@ def test_replace_bytes(self, frame_or_series):
         expected = obj.copy()
         obj = obj.replace({None: np.nan})
         tm.assert_equal(obj, expected)
+
+    @pytest.mark.parametrize(
+        "s, to_replace, value, expected",
+        [
+            (DataFrame([1]), np.array([1.0]), [0], DataFrame([0])),
```

Comment:
May want to test this for different container types for `to_replace` rather than only `np.array`

Reply:
I added lists too. Should I add more types?